script.js

```diff
@@ -1,121 +1,138 @@
+//Analog and Digital Clocks
+setInterval(drawClock, 1000); 
+digitalClock();  
 
+//Constants
+var borderWidthRatio = 0.1, innerCircleRatio = 0.1;
+var borderInnerRadiusRatio = 0.95;
+var borderOuterRadiusRatio = 1.05;
+var NUM_ON_CLOCK = 12;

```

Comment:
constants you should use const, and you should have these alphabetical and also with the CAPS_UNDERSCORE_CASE.

Read about const vs var and about hoisting of variables in Javascript,

also you can declare them in a const {} block